brain/circuit.cpp

```diff
@@ -275,13 +277,13 @@ class MorphologyLoader
         const size_t end = somaID == sections.size() - 1
                                ? points.size()
                                : sections[somaID + 1][0];
-        Vector3f centroid;
+        glm::vec3 centroid;
         for (size_t i = start; i != end; ++i)
-            centroid += points[i].get_sub_vector<3, 0>();
+            centroid += glm::vec3(points[i]);
```

Comment:
Not sure this is right.

Reply:
The get_sub_vector<Length, Start> returns a vector of Length whose coefficents are taken from the buffer starting at Start. So it takes the 3 first components of the 4 component vector. The constructor for vec3 does the same.